src/models/overdraft.py

```diff
@@ -9,6 +9,7 @@
 from web3 import Web3
 from src.abis.load import overdraftsmanager
 from src.models.accounting_period import AccountingPeriod
+from src.config import OverdraftConfig
```

Comment:
Is the models the right place to have such functionality? Will have another look at this